libzeth/tests/circuits/note_test.cpp

```diff
@@ -12,28 +12,35 @@
 #include "libzeth/core/utils.hpp"
 
 #include <gtest/gtest.h>
+#include <memory>
 
 using namespace libzeth;
 
-typedef libzeth::ppT ppT;
+using ppT = libzeth::ppT;
 
 // Should be alt_bn128 in the CMakeLists.txt
-typedef libff::Fr<ppT> FieldT;
+using FieldT = libff::Fr<ppT>;
 
 // We use our hash functions to do the tests
-typedef BLAKE2s_256<FieldT> HashT;
-typedef MiMC_mp_gadget<FieldT> HashTreeT;
-static const size_t TreeDepth = 4;
+using HashT = BLAKE2s_256<FieldT>;
+using HashTreeT = MiMC_mp_gadget<FieldT>;
+static const size_t tree_depth = 4;
 
 namespace
 {
 
+template<typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args &&... args)
```

Comment:
Manual addition since `std::make_unique` isn't in c++11, but in c++14. As such, I added these to make sure that the entire code was using functionalities from the cpp standard 11 since this is the one documented in the cmake config

Reply:
Why is the unique_ptr used at all?  It seems like the merkle tree can just be created on the stack so it's cleaned up automatically:
```
merkle_tree_field<FieldT, HashTreeT> test_merkle_tree(TreeDepth);
```
I may have misunderstood, but it seems unnecessary to use a smart pointer here.